test/validationRules/built-in.js

```diff
@@ -162,7 +162,7 @@ const validatorCases = {
     errors: [
       {
         message:
-          'Field "sum" argument "b" of type "Int!" is required but not provided.',
+          /Field "sum" argument "b" of type "Int!" is required/,
```

Comment:
Make these regexes so they pass in all versions of `graphql` that we test (the error messages have slight variations depending on the version).